XamarinCommunityToolkit/Views/MediaElement/MediaElement.shared.cs

```diff
@@ -205,27 +220,33 @@ void OnSourcePropertyChanged(MediaSource oldvalue, MediaSource newvalue)
 			InvalidateMeasure();
 		}
 
-		static void OnSourcePropertyChanging(BindableObject bindable, object oldvalue, object newvalue) => ((MediaElement)bindable).OnSourcePropertyChanging((MediaSource)oldvalue, (MediaSource)newvalue);
+		static void OnSourcePropertyChanging(BindableObject bindable, object oldvalue, object newvalue) =>
+			((MediaElement)bindable).OnSourcePropertyChanging((MediaSource)oldvalue);
 
-		void OnSourcePropertyChanging(MediaSource oldvalue, MediaSource newvalue)
+		void OnSourcePropertyChanging(MediaSource oldvalue)
 		{
 			if (oldvalue == null)
 				return;
 
 			oldvalue.SourceChanged -= OnSourceChanged;
 		}
+
+		static bool ValidateVolume(BindableObject o, object newValue)
+		{
+			var d = (double)newValue;
+
+			return d >= 0.0 && d <= 1.0;
+		}
 	}
 
-	[EditorBrowsable(EditorBrowsableState.Never)]
-	public class SeekRequested : EventArgs
+	class SeekRequested : EventArgs
```

Comment:
The XF version that we target has this same Class, so for now I'll keep it here. When we upgrade the XF version for the project we can refactor this easily.